newamericadotorg/assets/react/surveys-homepage/components/TeaserListing.js

```diff
@@ -279,7 +279,7 @@ class TeaserListing extends Component {
                   />
                 )}
               </div>
-              <div className="col-sm-3 teaser-listing__item-date">
+              <div className="col-sm-3 col-md-2 teaser-listing__item-date">
```

Comment:
Need to confirm date does not break on all screen widths

Reply:
see above